docs/search_index.json

```diff
@@ -1,3 +1,3 @@
 [
-["index.html", "Wildlife Management and Invasive Species (INBO) This homepage provides an overview of all interactive web versions of reports made by the Wildlife Management and Invasive Species group at the Research Institute for Nature and Forest (INBO, Flanders, Belgium).", " Wildlife Management and Invasive Species (INBO) This homepage provides an overview of all interactive web versions of reports made by the Wildlife Management and Invasive Species group at the Research Institute for Nature and Forest (INBO, Flanders, Belgium). Contact information faunabeheer@inbo.be Welcome to our interactive reports homepage, which, clearly, is still under construction. The reports embedded are however all finished. Our current (limited) list of interactive reports is: Beavermonitoring Enjoy reading! "]
+["index.html", "Wildlife Management and Invasive Species (INBO) This homepage provides an overview of all interactive web versions of reports made by the Wildlife Management and Invasive Species group at the Research Institute for Nature and Forest (INBO, Flanders, Belgium).", " Wildlife Management and Invasive Species (INBO) This homepage provides an overview of all interactive web versions of reports made by the Wildlife Management and Invasive Species group at the Research Institute for Nature and Forest (INBO, Flanders, Belgium). Contact information faunabeheer@inbo.be Welcome to our interactive reports homepage, which, clearly, is still under construction. The reports embedded are however all finished. Our current (limited) list of interactive reports is: Beaver telemetry Beavermonitoring Enjoy reading! "]
 ]
```

Comment:
maybe add an "&" here

Reply:
Beaver telemetry & Beavermonitoring Enjoy reading!

Reply:
Niet nodig, die staan op homepage netjes onder elkaar in lijst toch?

Reply:
Maar een manier om dat te weten, ik merge!